### ISSUE TITLE
abou test

### ISSUE THREAD

**Issue by liu-tianxiang** [mrFahrenhiet/CrackSegmentationDeepLearning#4]
Hello, I used your pre-training parameters to test a few 1124 pictures, and found that the effect is very poor. Does your method handle the data set ?   and your result graph is not separated separately. Can you re-test so that the result set only has the predicted picture?

**Comment by mrFahrenhiet** [comment]
Which dataset did you use? I tested my work on crack500 test dataset

**Comment by liu-tianxiang** [comment]
Thanks for your reply, I retuned the code. I downloaded the logs_finished_test.csv file in your Google Cloud, and averaged the data in the file, and found that test_f1 is 75.80, not 0.775 in the Readme. Do you directly average this PRF indicator?

**Comment by 3mmmouth** [comment]
I found in the Colab Notebook link (Base Code.ipynb) that the model_best IoU is 0.621 , not 0.663 in the Readme.

Validation Epoch: [1/1] Loss: 0.22809680925145268 Dice: 0.7719957045813048 IoU: 0.6214362141076356 Accuracy: 0.9749175004687598 Precision: 0.7633106557924129 Recall: 0.7907577502769931 F1: 0.7750518841675592: 100%|██████████| 71/71 [00:06<00:00, 10.55it/s]

**Comment by liu-tianxiang** [comment]
Because the final result of this paper is to produce a The csv file is not a specific value, so it is difficult for me to find the value in readme. Have you reproduced this code? I want to know how you calculate the test results, and whether the average value is directly taken from the. csv file.

**Comment by mrFahrenhiet** [comment]
Hi first of all sorry for the late reply. I must apologize for the discrepancy in the readme, those results were taken from my paper on crack segmentation and not this experiment. I will update the actual results in the readme when I have time. To answer your question, I used the average of F1 Scores on the test split of the Crack500 dataset. The CSV you see represents the metrics of a single image from the Crack500 test dataset.

**Comment by liu-tianxiang** [comment]
I know that csv is the value of 1124 images in the test set, but I used the pth test you gave, but I couldn't achieve the effect of the validation set you published. Can you check whether this pth is the best pth in your paper?

**Comment by liu-tianxiang** [comment]
If you check correctly, I don't know why I ran the pth test you generated so badly. Can you send me the dataset you used? My email is 2279566500@qq.com , thank you
